### PR TITLE
EZP-30909: Form created On the Fly should take full modal view

### PR DESCRIPTION
**Pull request by GrabowskiM**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-30909
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no

Removed footer from form on the fly
https://github.com/ezsystems/ezplatform-form-builder/pull/206

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/modules/universal-discovery/content.create.tab.module.js [hunk 1]
```diff
@@ -63,7 +78,7 @@ const ContentCreataTabModule = () => {
     const confirmLabel = Translator.trans(/*@Desc("Confirm")*/ 'content_create.confirm.label', {}, 'universal_discovery_widget');
 
     return (
-        <div className="m-content-create">
+        <div className={`m-content-create ${footerVisible ? 'm-content-create--footer-visible' : ''}`}>
```
Comment (dew326) [inline src/modules/universal-discovery/content.create.tab.module.js]: We have helper `createCssClassNames` please use it

File: src/modules/universal-discovery/content.create.tab.module.js [hunk 2]
```diff
@@ -31,6 +32,20 @@ const ContentCreataTabModule = () => {
     const [selectedLocations, dispatchSelectedLocationsAction] = useContext(SelectedLocationsContext);
     const iframeUrl = generateIframeUrl(contentOnTheFlyData);
     const iframeRef = createRef();
+
```
Comment (dew326) [inline src/modules/universal-discovery/content.create.tab.module.js]: ****
```suggestion
```

**Review by dew326 — CHANGES_REQUESTED** [review]

**Review by OstafinL — APPROVED** [review]

File: src/modules/universal-discovery/content.create.tab.module.js [hunk 3]
```diff
@@ -59,11 +60,22 @@ const ContentCreataTabModule = () => {
             });
         }
     };
+    const hideFooter = () => setFooterVisible(false);
+    const showFooter = () => setFooterVisible(true);
     const cancelLabel = Translator.trans(/*@Desc("Cancel")*/ 'content_create.cancel.label', {}, 'universal_discovery_widget');
     const confirmLabel = Translator.trans(/*@Desc("Confirm")*/ 'content_create.confirm.label', {}, 'universal_discovery_widget');
+    useEffect(() => {
```
Comment (dew326) [inline src/modules/universal-discovery/content.create.tab.module.js:90]:
```suggestion

    useEffect(() => {
```

**Review by dew326 — APPROVED** [review]

**Review by tomaszszopinski — APPROVED** [review]
QA Approved on eZ Platform EE v3 master with diff.

**Pull request merged**